### PR TITLE
Remove uses of deprecated APIs

### PR DESCRIPTION
**Pull request by peastman**
This removes a lot of uses of deprecated APIs that will be going away in TensorFlow 2.  Lots more still to do, but it will be easier if I split it between several PRs.  This is the first batch.

### CODE REVIEW

**Comment by peastman** [comment]
Some tests are failing because the signature for `tf.nn.dropout` changed between 1.12 and 1.13.  What do you think about bumping the development branch to TF 1.13, so we can use the non-deprecated argument?

**Comment by VIGS25** [comment]
I would like the bumping to TF 1.13. I have suggested the same in my GSOC application, because of availability of a particular functionality of keras, only with TF 1.13.

**Comment by peastman** [comment]
This should be ready to merge.
